src/_gettsim_tests/test_data/renten_anspr.csv

```diff
@@ -22,5 +22,5 @@ hh_id,tu_id,p_id,bruttolohn_m,wohnort_ost,alter,weiblich,jahr,geburtsjahr,geburt
 21,21,21,8000.0,TRUE,59,FALSE,2018,1952,1,41,0.8587,35.2067,,1.79167737,36.99837737,,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,60.0,0.0,0.0,0.0,0.0,,65.5,4650,,from eastern Germany: earns less Entgeltpunkte for same wage because Beitragsbemessungsgrenze is lower,https://www.ihre-vorsorge.de/rechner/rentenbeginnrechner.html
 22,22,22,8000.0,TRUE,59,FALSE,2018,1947,1,41,0.8587,35.2067,,1.79167737,36.99837737,,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,60.0,0.0,0.0,0.0,0.0,,65.08333333,4650,,from eastern Germany: earns less Entgeltpunkte for same wage because Beitragsbemessungsgrenze is lower,https://www.ihre-vorsorge.de/rechner/rentenbeginnrechner/ergebnis.html
 23,23,23,0.0,FALSE,53,FALSE,2018,1958,1,35,0.8875,31.0625,,0,31.0625,,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,60.0,0.0,0.0,0.0,0.0,,66,5600,,,https://www.deutsche-rentenversicherung.de/SiteGlobals/Forms/RentenBeginnUndHoehenRechner/Rentenbeginn/rentenbeginnrechner_form.html?resourceId=fd80eb35-41ae-49ec-a76d-fae25f0fb182&input_=26d0ef0f-0f0e-4a04-b1db-4e7dbbba7228&pageLocale=de&emailText=&Geburtsdatum=01.01.1958&Schwerbehindert=Nein&Schwerbehindert.GROUP=1&Bergbau=Nein&Bergbau.GROUP=1&submit=Berechnen
-24,24,24,1500.0,FALSE,72,TRUE,2010,1938,12,35,0.8968,22.42,,0.577960442,22.99796044,,0.0,0.0,9.0,0.0,0.0,0.0,0.0,60.0,15.0,420.0,0.0,0.0,108.0,10.0,,60,5500,,Beispiel: Erna RÃ¼stig (Altersrente fÃ¼r Frauen),https://rentenbescheid24.de/die-altersrente-fuer-langjaehrig-versicherte/
-25,25,25,1500.0,FALSE,54,TRUE,2010,1948,12,35,0.8968,22.42,,0.577960442,22.99796044,,0.0,0.0,9.0,0.0,0.0,0.0,0.0,60.0,15.0,420.0,0.0,0.0,108.0,10.0,,65,5500,,Beispiel: Abwandlung Erna RÃ¼stig (Altersrente fÃ¼r langjÃ¤hrig Versicherte),https://rentenbescheid24.de/die-altersrente-fuer-langjaehrig-versicherte/
+24,24,24,1500.0,FALSE,71,TRUE,2010,1938,12,35,0.8968,22.42,,0.577960442,22.99796044,,0.0,0.0,9.0,0.0,0.0,0.0,0.0,60.0,15.0,420.0,0.0,0.0,108.0,10.0,,60,5500,,Beispiel: Erna RÃ¼stig (Altersrente fÃ¼r Frauen),https://rentenbescheid24.de/die-altersrente-fuer-langjaehrig-versicherte/
```

Comment:
Where do these changes come from?

Reply:
The first one just for symmetry of ages (no difference in results) the second one actually fixes age and the relevant year, they were off otherwise. LMK if I had a mistake in my thinking there, @TeBackh, would be great if you could double-check line 26 here.

Reply:
I don't fully understand which symmetry you are referring to, but I am fine with it.

Reply:
The only change is the age -- Person born 1938 is assumed to be 71 years of age at whatever point we are evaluating in 2010. Same as next line.